backend/putio/fs.go

```diff
@@ -302,7 +302,7 @@ func (f *Fs) createUpload(ctx context.Context, name string, size int64, parentID
 		if err != nil {
 			return false, err
 		}
-		if err := checkStatusCode(resp, 201); err != nil {
+		if err := checkStatusCode(resp, []int{201}); err != nil {
```

Comment:
...This can then become (ie it doesn't need to change)

    checkStatusCode(resp, 201);